### DISCUSSION TITLE
Coordinates of bounding boxes

### DISCUSSION THREAD

**Discussion by bernardohenriques** [ultralytics/ultralytics#2478] [General]
Hello, 

I am wondering if there is a way to get the coordinates (center, width and height) of the bounding boxes when the object detection algorithm is running and detects an object.

Thank you.

**Comment by joejep** [comment]
I am also look for a way to get the bounding boxes, class probabilities and image features of the images passed

**Reply by glenn-jocher** [reply]
@eiyike123 hello!

Yes, when you run the object detection with YOLOv8, the model outputs the coordinates of the bounding boxes, which include the center coordinates, width, and height for each detected object. Additionally, it provides the class probabilities for each detection.

For image features, YOLOv8 models output a feature vector for each detected object, which can be accessed during prediction. These features can be used for various tasks, such as similarity comparison or object re-identification.

To obtain all this information, you can use the Predict mode, which will give you the detections and the associated metadata. For further details on how to extract and work with these outputs, please refer to the Predict section in our documentation. 📚

Happy detecting! 🚀
